students/vvinodh/Lesson3/mailroom.py

```diff
@@ -0,0 +1,101 @@
+donors = [
+        ["William Gates, III", 653784.49, 2, 326892.24],
+        ["Mark Zuckerberg", 16396.10, 3, 5465.37],
+        ["Jeff Bezos", 877.33, 1, 877.33],
+        ["Paul Allen", 708.42, 3, 236.14],
+        ]
+
+
+def menu():
+    quit_program = False
+    while not quit_program:
+
+        print("\n------------------ Main Menu ------------------\n")
+        print("Choose an action:\n")
+        print("Press 1 to send a Thank You\n")
+        print("Press 2 to create a report\n")
+        print("Press 3 to quit the program\n")
+        choice = input("-> ")
+
+        if choice == '1':
+            thankyou()
+        elif choice == '2':
+            report()
+        elif choice == '3':
+            quit_program = True
+            break
```

Comment:
no need for both `break` AND boolean flag (`quit_program`)

recommended to change to `while True` and keep `break` statement.